packages/devextreme/testing/testcafe/tests/accessibility/tabs.ts

```diff
@@ -0,0 +1,57 @@
+import url from '../../helpers/getPageUrl';
+import { clearTestPage } from '../../helpers/clearPage';
+import { testAccessibility, Options, Configuration } from '../../helpers/accessibility/testAccessibility';
+import { Item } from '../../../../js/ui/tabs.d';
```

Comment:
minor
```suggestion
import { Item } from '../../../../js/ui/tabs';
```

Reply:
Fixed

Reply:
No it's not :D

Reply:
Oh, but i got why you wrote this
eslint analyze only .js file and can't find proper type in .d.ts. I think we need to remove this rule, let's do it separately